frontend/src/modules/fluenteditor/components/RichTranslationForm.js

```diff
@@ -466,14 +482,19 @@ export default class RichTranslationForm extends React.Component<EditorProps> {
             return null;
         }
 
+        let attributeLabel = null;
         if (!label) {
             label = 'Value';
         }
+        else {
+            attributeLabel = label;
```

Comment:
When `label` is truthy, both `label` and `attributeLabel` have the same string value, which I think is a bit misleading. At least it was for me. I don't have a good suggestions about how to fix this, however. Perhaps find a better name for `label`? Or add a comment about its purpose?